src/mailadm/data/sysconfig/mailadm-prune.service

```
@@ -3,7 +3,8 @@ Description=Mailadm account pruning job
 After=network.target
 
 [Service]
-User=mailadm 
+User={mailadm_user}
```

```suggestion
User={mailadm_user}
Group={mailadm_user}
```

I think group needs to be set as well... isn't it the group permissions which make the difference here? Maybe one of us can try it out without Group, right now it's set on testrun.org.